packages/asktug/src/hackHeader.js

```diff
@@ -0,0 +1,53 @@
+import React, { useEffect, useState } from 'react';
+
+export const HackUserProfileSlot = () => {
+  const [width, setWidth] = useState(0);
+
+  const resetWidth = () => {
+    const el = document.querySelector('.d-header .panel');
+    if (el) {
+      setWidth((parseInt(getComputedStyle(el).width) ?? 0) + 16);
```

Comment:
Not quite sure about the compatibility of `getComputedStyle` on mobile devices. Might be all good.